src/js/contexts/AnnounceContext/stories/Polite.js

```diff
@@ -1,47 +1,67 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { useContext, useState } from 'react';
+
+import { Announce } from 'grommet-icons';
 import { grommet } from 'grommet/themes';
-import { AnnounceContext, Box, Grommet, Heading, Text } from 'grommet';
+import {
+  Anchor,
+  AnnounceContext,
+  Box,
+  Button,
+  Grommet,
+  Heading,
+  Paragraph,
+  Text,
+} from 'grommet';
 
-const Announcer = ({ announce, message, mode, role }) => {
-  React.useEffect(() => {
-    const timeout = 3000;
-    announce(message, mode, timeout);
-  }, [announce, message, mode]);
+const message = `Thank you for clicking the Announce Button, 
+this announcement is currently being broadcast.`;
 
+const Announcer = ({ mode, role }) => {
+  const announce = useContext(AnnounceContext);
+  const [show, setShow] = useState(false);
   return (
-    <Text align="center" role={role} aria-live={mode}>
-      {message}
-    </Text>
-  );
-};
-
-Announcer.propTypes = {
-  announce: PropTypes.func.isRequired,
-  message: PropTypes.string,
-  mode: PropTypes.string,
-  role: PropTypes.string,
-};
+    <Box align="center" gap="medium">
+      <Button
+        label="Announce"
+        icon={<Announce />}
+        a11yTitle="Announce button"
+        reverse
+        onClick={() => {
+          setShow(true);
+          announce(message, mode);
+        }}
+      />
 
-Announcer.defaultProps = {
-  message: 'Here is a simple announcement. This will soon disappear',
-  mode: 'polite',
-  role: 'log',
+      {show && (
+        <Text role={role} aria-live={mode}>
+          {message}
+        </Text>
+      )}
+    </Box>
+  );
 };
 
-const AnnounceContextComponent = props => (
+export const Polite = () => (
   <Grommet theme={grommet} full>
-    <Box justify="center" align="center" background="brand" fill>
+    <Box justify="center" align="center" background="brand" fill gap="medium">
       <Heading>Welcome to announcement section</Heading>
-      <AnnounceContext.Consumer>
-        {announce => <Announcer announce={announce} {...props} />}
-      </AnnounceContext.Consumer>
+      <>
+        <Paragraph textAlign="center">
+          This component will demonstrate a screen reader accessibility feature
+          via turning on the
+          <Anchor
+            label=" turning on the Screen reader mode."
+            href="https://www.codecademy.com/articles/how-to-setup-screen-reader#:~:text=(OS%20X)%20VoiceOver,Command%2DF5%20turns%20it%20off."
+          />{' '}
+          An The announcement will be broadcast once clicking on the announce
+          Button, turn on your clients Screen Reader mode to listen:
+        </Paragraph>
+      </>
+      <Announcer mode="polite" role="log" />
```

Comment:
I know we need to be a separate component to simplify access to the AnnounceContext. But, I think we should call it something else, and maybe put the Paragraph inside it, to make it clear you can embed it inside your existing content and don't need to create a separate "Announce" component to get announcements.